Sources/TuistKit/Services/Cache/CacheProfileResolver.swift

```diff
@@ -1,36 +1,64 @@
 import Foundation
 import TuistGraph
+import TuistSupport
 
 struct CacheProfileResolver {
-    enum ResolvedCacheProfile: Equatable {
-        case defaultFromTuist(TuistGraph.Cache.Profile)
-        case defaultFromConfig(TuistGraph.Cache.Profile)
-        case selectedFromConfig(TuistGraph.Cache.Profile)
-        case notFound(profileName: String, availableProfiles: [String])
+    enum Error: FatalError, Equatable {
```

Comment:
to stay consistent with the rest of the codebase, I'd name this explicitly `CacheProfileResolverError`